### ISSUE TITLE
Template strings inside JSX error.

### ISSUE THREAD

**Issue by wavded** [eslint/eslint#2110]
Using latest eslint, with `es6` mode turned on and given this code, template strings lint fine until I add them to a JSX block.

```js
import React from 'react'

let foo = 'foo'
let baz = `baz${foo}` // no error

React.renderToString(
  <div
    label={`Bar (${baz})`} // Unexpected token ILLEGAL
    disabled={true}
    checked={true} />
)
```

**Comment by wavded** [comment]
With the greater code context (where I pulled this example from), I get this error:

```
clientapp/components/sidebar.js
  130  -96  error  Unexpected quasi
                disabled={true}
                checked={true} />
```

**Comment by BenoitZugmeyer** [comment]
Reported here: eslint/espree#95

**Comment by wavded** [comment]
Ahh thanks, should have checked there as well to see if someone reported.

**Comment by mikesherov** [comment]
@wavded what is the code that produces that error?

**Comment by nzakas** [comment]
The Espree bug has been fixed and a patch release published: https://github.com/eslint/espree/releases/tag/v1.12.1

You should be able to `npm i eslint@latest --save-dev` to get the changes.

**Comment by wavded** [comment]
Thanks @nzakas
